### PR TITLE
Integration with Openshift

### PR DESCRIPTION
**Pull request by ChrisGadek1**

### CODE REVIEW

File: nginx.conf [hunk 1]
```diff
@@ -1,17 +1,7 @@
 # generated 2023-10-17, Mozilla Guideline v5.7, nginx 1.17.7, OpenSSL 1.1.1k, intermediate configuration
 # https://ssl-config.mozilla.org/#server=nginx&version=1.17.7&config=intermediate&openssl=1.1.1k&guideline=5.7
 server {
-    listen 80 default_server;
```
Comment (grzanka) [inline nginx.conf:4]: Why the port of UI was changed to 3000 ? That will break standard deployment

Comment (ChrisGadek1) [reply nginx.conf]: I tried and it seems it's impossible to run on port 80 without root. To make it work on non-root user I need to run it on some other port

Comment (grzanka) [reply nginx.conf:4]: What about HTTPS ? You disabled it completely. Its needed for standard deploy

Comment (ChrisGadek1) [reply nginx.conf:4]: Maybe to resolve it, there is a need to create two separate nginx configurations. What I know is impossible to expose port 80 on non-root container

Comment (grzanka) [reply nginx.conf:4]: Port 80 is not crucial, it was used only for some internal development. We can get rid of it. Most important is to have working solution with SSL.

**Comment by grzanka** [comment]
Postponing till later

**Pull request closed without merging**